momento-sdk/src/intTest/java/momento/sdk/SimpleCacheDataPlaneBlockingTest.java

```diff
@@ -106,48 +102,64 @@ void itemDroppedAfterTtlExpiresWithTracing() throws Exception {
   }
 
   @Test
-  public void badTokenThrowsAuthenticationException() {
-    String badToken =
-        "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJpbnRlZ3JhdGlvbiIsImNwIjoiY29udHJvbC5jZWxsLWFscGhhLWRldi5wcmVwcm9kLmEubW9tZW50b2hxLmNvbSIsImMiOiJjYWNoZS5jZWxsLWFscGhhLWRldi5wcmVwcm9kLmEubW9tZW50b2hxLmNvbSJ9.gdghdjjfjyehhdkkkskskmmls76573jnajhjjjhjdhnndy";
-    SimpleCacheClient target =
-        SimpleCacheClient.builder(badToken, DEFAULT_ITEM_TTL_SECONDS).build();
-    assertThrows(AuthenticationException.class, () -> target.get(cacheName, ""));
+  public void badTokenReturnsAuthenticationError() {
+    final String badToken =
+        "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJpbnRlZ3JhdGlvbiIsImNwIjoiY29udHJvbC5jZWxsLWFscGhhLWRldi5"
+            + "wcmVwcm9kLmEubW9tZW50b2hxLmNvbSIsImMiOiJjYWNoZS5jZWxsLWFscGhhLWRldi5wcmVwcm9kLmEub"
+            + "W9tZW50b2hxLmNvbSJ9.gdghdjjfjyehhdkkkskskmmls76573jnajhjjjhjdhnndy";
+    try (final SimpleCacheClient client =
+        SimpleCacheClient.builder(badToken, DEFAULT_ITEM_TTL_SECONDS).build()) {
+
+      final CacheGetResponse response = client.get(cacheName, "");
+      assertThat(response).isInstanceOf(CacheGetResponse.Error.class);
+      assertThat(((CacheGetResponse.Error) response).exception())
+          .isInstanceOf(AuthenticationException.class);
+    }
   }
 
   @Test
-  public void nonExistentCacheNameThrowsNotFoundOnGetOrSet() {
-    SimpleCacheClient target =
-        SimpleCacheClient.builder(authToken, DEFAULT_ITEM_TTL_SECONDS).build();
-    String cacheName = UUID.randomUUID().toString();
+  public void nonExistentCacheNameReturnsErrorOnGetOrSet() {
+    try (final SimpleCacheClient client =
+        SimpleCacheClient.builder(authToken, DEFAULT_ITEM_TTL_SECONDS).build()) {
+      final String cacheName = UUID.randomUUID().toString();
 
-    assertThrows(NotFoundException.class, () -> target.get(cacheName, ""));
+      final CacheGetResponse response = client.get(cacheName, "");
+      assertThat(response).isInstanceOf(CacheGetResponse.Error.class);
+      assertThat(((CacheGetResponse.Error) response).exception())
+          .isInstanceOf(NotFoundException.class);
 
-    assertThrows(NotFoundException.class, () -> target.set(cacheName, "", "", 10));
+      assertThrows(NotFoundException.class, () -> client.set(cacheName, "", "", 10));
+    }
   }
 
   @Test
   public void setGetDeleteWithByteKeyValuesMustSucceed() {
-    byte[] key = {0x01, 0x02, 0x03, 0x04};
-    byte[] value = {0x05, 0x06, 0x07, 0x08};
-    SimpleCacheClient cache =
-        SimpleCacheClient.builder(authToken, DEFAULT_ITEM_TTL_SECONDS).build();
-    cache.set(cacheName, key, value, 60);
+    final byte[] key = {0x01, 0x02, 0x03, 0x04};
+    final byte[] value = {0x05, 0x06, 0x07, 0x08};
```

Comment:
I recommend generating a UUID as a byte array (similar to the way you did for strings). That way if we run the tests in quick succession we won't collide with past test data.

Reply:
This class is being deleted in the next pr so I'm leaving this for now.